hack/testing/prep-host

```diff
@@ -10,6 +10,10 @@ OS_ANSIBLE_REPO=${OS_ANSIBLE_REPO:-https://github.com/openshift/openshift-ansibl
 OS_ANSIBLE_BRANCH=${OS_ANSIBLE_BRANCH:-master}
 OS_ANSIBLE_DIR=$WORKDIR/openhift-ansible
 
+if [ -n PUDDLE_URL ]; then
```

Comment:
you should be able to install ansible-2.2.0.0 using pip `pip install -Iv ansible==2.2.0.0`